src/dsp_tools/error/exceptions.py

```diff
@@ -65,9 +69,20 @@ class InvalidGuiAttributeError(BaseError):
     """This error is raised when a invalid gui-attribute is used."""
 
 
-class UnexpectedApiResponseError(BaseError):
+class FatalNonOkApiResponseCode(BaseError):
     """This error is raised when the API gives an unexpected response, that we cannot anticipate and handle cleanly."""
 
+    def __init__(self, request_url: str, status_code: int, response_text: str) -> None:
+        resp_txt = response_text[:200] if len(response_text) > 200 else response_text
+        msg = (
+            f"We currently do not support the following API response code for this request. "
+            f"Please contact the dsp-tools development team (at info@dasch.swiss) to handle this bug.\n"
+            f"Status code: {status_code}\n"
+            f"Request URL: {request_url}\n"
+            f"Original Response: {resp_txt}"
```

Comment:
In order for this to work, you need to add this: `from dsp_tools.config.logger_config import LOGGER_SAVEPATH`:

```suggestion
            f"We currently do not support the following API response code for this request.\n"
            f"Status code: {status_code}\n"
            f"Request URL: {request_url}\n"
            f"Original Response: {resp_txt}\n"
            f"Please contact info@dasch.swiss with the log file at {LOGGER_SAVEPATH}."
```